PR for SRVLOGIC-300, SRVLOGIC-190, SRVLOGIC-188, SRVLOGIC-192: Document "Serverless Logic" parent directory for Openshift Serverless Logic docs. (#76490)
openshift/openshift-docs: modules/serverless-logic-workflow-changing-base-builder-image.adoc

// Module included in the following assemblies:
//
// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc

:_content-type: PROCEDURE
[id="serverless-logic-workflow-changing-base-builder-image"]
= Changing the base builder image

> You can modify the default builder image used by the {ServerlessLogicOperatorName} by editing the `sonataflow-operator-builder-config` ConfigMap.

Reviewer: ```suggestion
You can modify the default builder image used by the {ServerlessLogicOperatorName} by editing the `logic-operator-rhel8-builder-config` ConfigMap.
```

.Prerequisites

* {ServerlessLogicOperatorName} is installed on your cluster.
* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
* You have created your {ServerlessLogicProductName} project
* You have installed the OpenShift CLI `(oc)`.

.Procedure

> . Edit the `sonataflow-operator-builder-config` ConfigMap.

Reviewer: ```suggestion
. Edit the `logic-operator-rhel8-builder-config` ConfigMap.
```

+
[source,terminal]
----

> $ oc edit cm/sonataflow-operator-builder-config -n sonataflow-operator-system

Reviewer: ```suggestion
$ oc edit cm/logic-operator-rhel8-builder-config -n openshift-serverless-logic
```

----

. Modify the dockerfile entry.
+
In your editor, locate the dockerfile entry and change the first line to the desired image.
+
.Example
[source,yaml]
----
data:
  Dockerfile: |

>     FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8:1.32.0

Reviewer: ```suggestion
    FROM registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.33.0
```

// Module included in the following assemblies:
//
// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc

:_content-type: PROCEDURE
[id="serverless-logic-workflow-changing-base-builder-image"]
= Changing the base builder image

You can modify the default builder image used by the {ServerlessLogicOperatorName} by editing the `sonataflow-operator-builder-config` ConfigMap.

.Prerequisites

* {ServerlessLogicOperatorName} is installed on your cluster.
* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
* You have created your {ServerlessLogicProductName} project
* Install the OpenShift CLI `(oc)`.

.Procedure

. Edit the `sonataflow-operator-builder-config` ConfigMap.
[source,terminal]
----
$ oc edit cm/sonataflow-operator-builder-config -n sonataflow-operator-system
----

. Modify the dockerfile entry.
+
In your editor, locate the dockerfile entry and change the first line to the desired image.
+
.Example
[source,yaml]
----
data:
  Dockerfile: |

>     FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8:1.32.0

Reviewer 2: ```suggestion
    FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8:1.32.0
```

Can you replace `1.32.0` with a variable to reflect the current version?

Reviewer: This is not 100% fine, use mine suggestion below please. We must use the builder related image, instead of the data-index related.

Author: Okay, noted!
